### ISSUE TITLE
Error when using accepts_nested_attributes_for and f.input

### ISSUE THREAD

**Issue by eduardordm** [heartcombo/simple_form#241]
Hi Guys!

I'll try to explain this best as I can (Maybe I had a beer or two :D).

Thing is: If you have a Model that have accepts_nested_attributes_for in it AND I build that association in my NEW action I can only use f.association. f. input will stop working. 

<pre>
undefined method `gsub' for nil:NilClass

simple_form (1.3.1) lib/simple_form/inputs/string_input.rb:9:in `input'
simple_form (1.3.1) lib/simple_form/components/label_input.rb:9:in `label_input'
simple_form (1.3.1) lib/simple_form/inputs/base.rb:53:in `block in render'
simple_form (1.3.1) lib/simple_form/inputs/base.rb:51:in `each'
simple_form (1.3.1) lib/simple_form/inputs/base.rb:51:in `render'
simple_form (1.3.1) lib/simple_form/form_builder.rb:89:in `input'
</pre>

Long story short, that will not work:
(controller)

<pre>
  # GET /proposals/new
  # GET /proposals/new.xml
  def new
    @proposal = Proposal.new
    @proposal.build_client
  end
</pre>

(view)

<pre>
f.input :client, :as => :string
</pre>

If I don't build_client, will work. 

That always work (with or without build_client):

<pre>
 f.association :client 
</pre>

Thanks!!!

PS: If you ask why would I need an association input without a dropdown, simple: the dropdown will be loaded by jquery/ajax (and it's not a real option-list). I just need that input text to do it.
